turbo.json

```diff
@@ -4,6 +4,7 @@
   "pipeline": {
     "build": {
       "dependsOn": ["^build"],
+      "inputs": ["src", "package.json"],
```

Comment:
@srchase I couldn't get `yarn build` in the root to work without this. Builds would fail because the turbo cache would output a blank dist-cjs and dist-types folder for some packages. Is it possible some of the other turbo commands are causing bad values to be stored in the turbo cache?